.github/workflows/test-credentials.yml

```
@@ -0,0 +1,43 @@
+name: Test Credentials
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: test-credentials-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-gcp-credentials:
+    name: Test GCP credentials
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: google-github-actions/auth@v3
+        with:
+          credentials_json: ${{ secrets.TEST_GCP_CREDENTIALS }}
+      - uses: google-github-actions/setup-gcloud@v2
+      - run: gcloud config get-value project > /dev/null
```

I'd suggest echoing something to give a clear indication the command succeeded:

```suggestion
      - run: gcloud config get-value project > /dev/null && echo "GCP credentials test succeeded"
```